js_modules/dagit/packages/core/src/instance/BackfillUtils.tsx

```diff
@@ -0,0 +1,67 @@
+import {gql} from '@apollo/client';
+
+import {PYTHON_ERROR_FRAGMENT} from '../app/PythonErrorInfo';
+
+export const RESUME_BACKFILL_MUTATION = gql`
+  mutation resumeBackfill($backfillId: String!) {
```

Comment:
It might be nice to uppercase this so the type that is autogenerated is also capitalized